crates/chain/tests/test_tx_graph.rs

```diff
@@ -135,20 +135,20 @@ fn insert_txouts() {
     };
 
     // Check the resulting addition.
-    let additions = graph.determine_additions(&update);
+    let changeset = graph.apply_update(update);
 
     assert_eq!(
-        additions,
-        Additions {
+        changeset,
+        ChangeSet {
             txs: [update_txs.clone()].into(),
             txouts: update_ops.into(),
             anchors: [(conf_anchor, update_txs.txid()), (unconf_anchor, h!("tx2"))].into(),
             last_seen: [(h!("tx2"), 1000000)].into()
         }
     );
 
-    // Apply addition and check the new graph counts.
-    graph.apply_additions(additions);
+    // Check the new graph counts.
```

Comment:
Doesn't this still apply the changeset?
```suggestion
    // Apply changeset and check the new graph counts.
```